### PR TITLE
Update sig meeting link

### PR DESCRIPTION
**Pull request by mwielgus**

### CODE REVIEW

**Comment by DirectXMan12** [comment]
I'd much prefer we just have a link to the SIG-Autoscaling page in the community repo, so there's only one source of this info, but I'm fine with this PR in the meantime.

**Comment by mwielgus** [comment]
I updated the community repo as well - https://github.com/kubernetes/community/pull/934

File: README.md [hunk 1]
```diff
@@ -21,8 +21,7 @@ Interested in autoscaling? Want to talk? Have questions, concerns or great ideas
 
 Please join us on #sig-autoscaling at https://kubernetes.slack.com/.
 Moreover, every Thursday we host a 30min sig-autoscaling meeting on
-https://hangouts.google.com/hangouts/_/google.com/sig-autoscaling at
-17:30 CEST/CET,  8:30 am PST/PDT.
+https://zoom.us/meeting/176352399 at 17:30 CEST/CET,  8:30 am PST/PDT.
```
Comment (DirectXMan12) [inline README.md]: whoops, needs to be `/j/` instead of `/meeting/`.

Comment (mwielgus) [inline README.md]: done

**Pull request merged**